Modules/Core/Common/include/itkMacro.h

```diff
@@ -776,7 +776,7 @@ compilers.
 
 #ifndef NDEBUG
 
-#  ifdef _POSIX_SOURCE
+#  ifdef __linux
```

Comment:
Good catch 👍 

Can we also make a PR to [the upstream](https://github.com/InsightSoftwareConsortium/ITK/blob/5dd95856d071e3f7c64f45afce3d27b366f9f682/Modules/Core/Common/include/itkMacro.h#L779) code?  We want this version to be as close as possible to the upstream.

Reply:
Done :) https://github.com/InsightSoftwareConsortium/ITK/pull/3369